### PR TITLE
Fix Android icons

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

File: gulpfile.js
Comment (KarateBrot) [inline gulpfile.js:394]:
```suggestion
  return gulp
```

Comment (haslinghuis) [reply gulpfile.js:394]: 4 space indentation is correct.

Comment (KarateBrot) [reply gulpfile.js:394]: Then the two lines below miss 4 intendations but no worries 😃

**Pull request merged**
